README.md

```diff
@@ -38,8 +38,8 @@ $ sbt
 > exit
 $ cd target/sbt-test/default-*/scripted
 $ sbt
-$ jetty:start
-$ browse # starts browser for you, or manually open http://localhost:8080 to verify
+> jetty:start
+-> executes browser, and opens `http://localhost:8080` to verify
```

Comment:
This line means to run `browse` task which is defined in the Scalatra sbt plugin. This task opens the application URL by the default web browser.
https://github.com/scalatra/scalatra-sbt/blob/03338c12b46fc6a609ffdb9fcb25cc24cf9db230/src/main/scala/org/scalatra/sbt/ScalatraPlugin.scala#L14

I think this description should be kept.